packages/types/src/availabilitySlot.ts

```diff
@@ -40,3 +46,38 @@ export type SubSlot = {
 };
 
 export type GeneralSlot = Slot | SubSlot;
+
+// API Payloads / Queries
+export type FindAvailabilitySlotsApiParams = {
```

Comment:
We use the term `Query` when namign the query params. 


```suggestion
export type FindAvailabilitySlotsApiQuery = {
```